modules/components/org.wso2.carbon.appfactory.provisioning.runtime/src/main/java/org/wso2/carbon/appfactory/provisioning/runtime/KubernetesRuntimeProvisioningService.java

```diff
@@ -550,6 +550,8 @@ public void addCustomDomain(Set<String> domains) throws RuntimeProvisioningExcep
                 kubClient.extensions().ingress().inNamespace(namespace.getMetadata().getName()).create(ing);
             }
         }
+
+        return false;
```

Comment:
Why we are returning false here? I cannot see a place where it returns true. Therefore this will be always false